web/angular-wallet/src/assets/particles.json

```diff
@@ -0,0 +1,108 @@
+{
+  "particles": {
+    "number": {
+      "value": 80,
+      "density": {
+        "enable": true,
+        "value_area": 400
+      }
+    },
+    "color": {
+      "value": "#ffffff"
+    },
+    "shape": {
+      "type": "circle",
+      "stroke": {
+        "width": 0,
+        "color": "#000000"
+      },
+      "polygon": {
+        "nb_sides": 5
+      },
+      "image": {
+        "src": "assets/images/logos/logo_blue.png",
```

Comment:
Was this intentional?

Reply:
I played around with an svg logo...but didn't work. So I removed the asset and put at least a valid path. So, yes, intentional